Lib/test/test_tcl.py

```diff
@@ -138,10 +138,14 @@ def testUnsetVarException(self):
 
     def get_integers(self):
         integers = (0, 1, -1, 2**31-1, -2**31, 2**31, -2**31-1, 2**63-1, -2**63)
-        # bignum was added in Tcl 8.5, but its support is able only since 8.5.8
-        if (get_tk_patchlevel() >= (8, 6, 0, 'final') or
-            (8, 5, 8) <= get_tk_patchlevel() < (8, 6)):
-            integers += (2**63, -2**63-1, 2**1000, -2**1000)
+        # bignum was added in Tcl 8.5, but its support is able only since 8.5.8.
+        # Actually it is determined at compile time, so using get_tk_patchlevel()
+        # is not reliable.
+        # TODO: expose full static version.
```

Comment:
I'm not sure that this TODO is useful.  Is there a need for the full static version for this specific test?

Reply:
The test will still fail if build Tkinter with 8.5 older than 8.5.8 or with alpha or beta of 8.6. Very unlikely.

I am planning to add named tuples (like `sys.version_info`) for static and dynamic versions. It is a part of large project which adds structured version for all external libraries.